.ci/ansible/tasks/setup_test_script.yml

```diff
@@ -133,8 +133,8 @@
     - "GITHUB_CHECK_SHA1=\"{{ lookup('env', 'GITHUB_CHECK_SHA1') or '' }}\""
     - "GITHUB_CHECK_REPO=\"{{ lookup('env', 'GITHUB_CHECK_REPO') or 'elastic-agent' }}\""
     - "ELASTIC_APM_GLOBAL_LABELS=\"{{ lookup('env', 'ELASTIC_APM_GLOBAL_LABELS') }}\""
-    - "LOG_LEVEL=\"TRACE\""
-    - "OP_LOG_LEVEL=\"TRACE\""
+    - "LOG_LEVEL=\"{{ lookup('env', 'LOG_LEVEL') or 'TRACE' }}\""
```

Comment:
This change is not related to docs, but found it while testing the docs. Closed #1941